utils/create-component/templates/css/packageJson.js

```diff
@@ -21,7 +21,14 @@ module.exports = (name, moduleName, description, unstable) => `
   "scripts": {
     "test": "echo \\"Error: no test specified\\" && exit 1",
     "build": "node ../../../${unstable ? '../' : ''}utils/css-build.js index.scss"
-  },
+  }, ${
+    public
+      ? `
+  "publishConfig": {
+    "access": "public"
+  },`
+      : ``
```

Comment:
Nit:
```suggestion
    public && `
  "publishConfig": {
    "access": "public"
  },`
```

would be cleaner

Reply:
I thought about that, but I was worried that the template literal will but a "false" in the json. I'll test that though.

Reply:
confirmed this.